pkg/agent/cniserver/server.go

```diff
@@ -44,6 +45,10 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsconfig"
 )
 
+const (
+	networkReadyTimeout = 30 * time.Second
```

Comment:
what's the default kubelet / container runtime timeout for CNI Add? could you add the info as a comment?